.golangci.yml

```diff
@@ -30,4 +30,4 @@ linters-settings:
     simplify: true
   misspell:
     ignore-words:
-      - transferer
+      - transferrer
```

Comment:
Just curious, does the linter still consider this a misspelling?

Reply:
nope all good now, removed